internal/testharness/pty/driver/driver.go

```
@@ -0,0 +1,231 @@
+func RunCommand(ctx context.Context, spec CommandSpec) (analyzer.Capture, error) {
+	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
+	if spec.Timeout > 0 {
+		var timeoutCancel context.CancelFunc
+		ctx, timeoutCancel = context.WithTimeout(ctx, spec.Timeout)
+		defer timeoutCancel()
+	}
+	if _, err := analyzer.NewDimensions(spec.Dimensions.Rows, spec.Dimensions.Cols); err != nil {
+		return analyzer.Capture{}, err
+	}
+	cmd := exec.Command(spec.Path, spec.Args...)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)